managed/services/telemetry/config.default.yml

```diff
@@ -1,54 +1,46 @@
 telemetry:
-# PMM - generic information
+  # PMM - generic information
   - id: PMMServerIAEnabled
     source: PMMDB_SELECT
-    query: ia->'enabled' AS ia_enabled FROM settings s, jsonb_extract_path(s.settings, 'ia') AS ia
+    query: (CASE WHEN ia->'enabled' = 'true' THEN '1' ELSE '0' END) AS ia_enabled FROM settings s, jsonb_extract_path(s.settings, 'ia') AS ia
     summary: "PMM Server Integration Alerting feature enabled/disabled"
     data:
-      - metric_name: "ia_enabled"
+      - metric_name: "pmm_server_ia_enabled"
         column: "ia_enabled"
 
   - id: PMMServerSTTEnabled
     source: PMMDB_SELECT
-    query: (CASE WHEN stt->'stt_disabled' = 'false' THEN 'true' ELSE 'false' END) AS stt_enabled FROM settings s, jsonb_extract_path(s.settings, 'sass') AS stt
+    query: (CASE WHEN stt->'stt_disabled' = 'false' THEN '1' ELSE '0' END) AS stt_enabled FROM settings s, jsonb_extract_path(s.settings, 'sass') AS stt
     summary: "PMM Server Security Thread Tool feature enabled/disabled"
     data:
-      - metric_name: "stt_enabled"
+      - metric_name: "pmm_server_stt_enabled"
         column: "stt_enabled"
 
-  - id: PMMServerBackupEnabled
+  - id: PMMServerBackupManagementEnabled
     source: PMMDB_SELECT
-    query: bm->'enabled' AS bm_enabled FROM settings s, jsonb_extract_path(s.settings, 'backup_management') AS bm
-    summary: "PMM Server Backup feature enabled/disabled"
+    query: (CASE WHEN backup_management->'enabled' = 'true' THEN '1' ELSE '0' END) AS backup_management_enabled FROM settings s, jsonb_extract_path(s.settings, 'backup_management') AS backup_management
+    summary: "PMM Server Backup Management feature enabled"
     data:
-      - metric_name: "backup_management_enabled"
-        column: "bm_enabled"
+      - metric_name: "pmm_server_backup_management_enabled"
+        column: "backup_management_enabled"
```

Comment:
```suggestion
        column: "pmm_server_backup_management_enabled"
```

Reply:
@rnovikovP column is mapped to select, metric_name will be used to send to the Portal
<img width="573" alt="image" src="https://user-images.githubusercontent.com/91831719/182419207-58688c4d-c5f4-4af2-ac4e-b64056c171c9.png">

Reply:
oh .. wrong line )